torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1180,6 +1182,28 @@ struct PythonPrintImpl {
           }
           stmt << *v;
         }
+
+        // scan backwards to find the first index where out args start
+        size_t out_start = num_schema_args - 1;
+        while (out_start >= necessary_args) {
+          auto arg = schema.arguments().at(out_start);
+          if (!arg.is_out()) {
+            break;
+          }
+          out_start--;
+        }
+
+        // print out args
+        for (size_t i = out_start + 1; i < schema.arguments().size(); i++) {
```

Comment:
Small suggestion:

```suggestion
        size_t out_start = num_schema_args;
        while (--out_start >= necessary_args) {
          auto arg = schema.arguments().at(out_start);
          if (!arg.is_out()) {
            break;
          }
        }

        // print out args
        for (size_t i = out_start; i < schema.arguments().size(); i++) {
```

That way ```out_start``` really ends up with the right index, and the for loop can start from that point without adjusting.

Reply:
thanks :) will do